src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -99,6 +99,26 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator with UserIn
             WorkspaceService.UnlockWorkspace(WorkspaceName(workspaceNamespace, workspaceName)))
         }
       }
+    } ~
+    path("workspaces" / Segment / Segment / "invites" ) { (workspaceNamespace, workspaceName) =>
+      post {
+        requireUserInfo() { userInfo =>
+          entity(as[WorkspaceACLUpdate]) { invite =>
```

Comment:
i was a little undecided on what these two endpoints should look like. three options i see are:

1) keep them as is
2) keep them almost as is, but make them take batches of invites to create/delete
3) make it similar to patch ACL where there is one endpoint and invites can be deleted by setting the invite's accessLevel to `NO ACCESS`

i thought of the 3rd option after the fact and think it would be nice to have that consistency